integration/src/test/resources/application-multizk.yaml

```diff
@@ -0,0 +1,91 @@
+owner:
```

Comment:
You don't need this configuration. You don't override any hermes-management instance in your integration test :)

Reply:
fixed